File: java/02/DayTwo.java
import java.util.*;
import java.math.*;

public class DayTwo {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        double mealCost = scan.nextDouble(); // original meal price
        int tipPercent = scan.nextInt(); // tip percentage
        int taxPercent = scan.nextInt(); // tax percentage
        scan.close();
        float totalCost;
      
        // Write your calculation code here.
        float tip = (float)mealCost * tipPercent/100;
        float tax = (float)mealCost * taxPercent/100;
      
        totalCost = (float)mealCost + tip + tax;
        // cast the result of the rounding operation to an int and save it as totalCost 
        int total = (int) Math.round(totalCost);
      
        // Print your result
        System.out.println("The total meal cost is "+total+" dollars.");
    }
}

File: perl/02/day-two.pl
#!/usr/local/bin/perl

use strict;
use warnings;

my $meal;
my $tip;
my $tax;
my $total;

alarm(1);
eval {
  local $SIG{ALRM} = sub { 
    $meal = 10.25; 
    die;
  };
  $meal = <STDIN>;
};

alarm(1);
eval {
  local $SIG{ALRM} = sub { 
    $tip = 17; 
    die;
  };
  $tip = <STDIN>;
};

alarm(1);
eval {
  local $SIG{ALRM} = sub { 
    $tax = 5; 
    die;
  };
  $tax = <STDIN>;
};

$tip = $meal * ($tip / 100);
$tax = $meal * ($tax / 100);

$total = $meal + $tip + $tax;

printf("The total meal cost is %.0f dollars.", $total); 
